api/v1beta1/common_types.go

```diff
@@ -41,7 +41,7 @@ type WatcherCommon struct {
 	// +kubebuilder:validation:Required
 	// MariaDB instance name
 	// Required to use the mariadb-operator instance to create the DB and user
-	DatabaseInstance string `json:"databaseInstance"`
+	DatabaseInstance *string `json:"databaseInstance"`
```

Comment:
yes I know this is draft but i am just trying to follow along here 
i am wondering/trying to understand why these are changed to references/pointers  with *string here

from quick search i could not find other examples where we are doing that https://github.com/search?q=org%3Aopenstack-k8s-operators+instance.Spec.DatabaseInstance&type=code

Reply:
i just saw the relevant comment above from abays about making these pointers... marking resolved thanks